File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.MessagingService.Models/get-notification-app-role-contexts_result.ts


import { NotificationAppRoleContext_Item } from './notification-app-role-context_item';
import { IGetNotificationAppRoleContexts_Result } from './iget-notification-app-role-contexts_result';

export class GetNotificationAppRoleContexts_Result implements IGetNotificationAppRoleContexts_Result {
  
	public NotificationID: number = 0;
	public ContextID: number = 0;
	public NotificationContextType: string = "";
	public Collection?: NotificationAppRoleContext_Item[];
  
}

File: GTTS/INL.MessagingService/Models/GetNotificationAppRoleContexts_Result.cs
using System.Collections.Generic;

namespace INL.MessagingService.Models
{
    public class GetNotificationAppRoleContexts_Result : IGetNotificationAppRoleContexts_Result
    {
        public long NotificationID { get; set; }
        public long ContextID { get; set; }
        public string NotificationContextType { get; set; }
        public List<NotificationAppRoleContext_Item> Collection { get; set; }
    }
}
